doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -30,11 +30,18 @@ Changelog
 
 The following sections provide detailed lists of changes by component.
 
-IDE, and tool support
-=====================
+IDE, OS, and tool support
+=========================
 
 * Added explicit mention of the :ref:`requirements_jlink` being required in the :ref:`installing_vsc` section of the installation page.
-* Updated the required `SEGGER J-Link`_ version to v7.94i.
+* Updated:
+
+  * The required `SEGGER J-Link`_ version to v7.94i.
```

Comment:
Would you like to convert the v7.94i to a shortcut in shortcut.txt? That way it could be also used by other documents.

Reply:
Not in changelog. Also, this PR is not about J-Link version, I'm just changing formatting.